README.md

```diff
@@ -29,13 +29,13 @@ Before you can actually start compiling and using the example applications pleas
 Your development setup has to meet the following prerequisites:
 
 * iOS 12 or later
-* Xcode 14.x, including Swift 5.7 or later
+* Xcode 15.x, including Swift 5.7 or later
 * Android 6 or later, with API level 23 -or-
 * Android 10 or later, with API level 29, for the biometric authenticator to work
 * Gradle 7.4 or later
 * Android Gradle Plugin `com.android.tools.build:gradle` 7.0.0 or later
 * Kotlin Gradle Plugin `org.jetbrains.kotlin:kotlin-gradle-plugin` 1.8.0 or later
-* Flutter SDK 3.x, stable channel
+* Dart SDK 3.1 or later
```

Comment:
Isn't is Dart `3.3.0`? This is the version that matches to Flutter `3.19.0`.

Reply:
You're right, copy/past issue :(